### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by natakazakova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ The Web Dashboard control can operate in `ViewerOnly` mode for unauthorized user
...
+You can limit access to sensitive information depending on the current user's ID. Every custom store/provider reads the `IHttpContextAccessor.HttpContext.User.Identity`. The standard [IHttpContextAccessor](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-context?view=aspnetcore-3.0) with dependency injection allow you to access the HTTP context and pass the retrieved user name to Dashboard providers listed above from the [MultiTenantDashboardConfigurator](./CS/Code/MultiTenantDashboardConfigurator.cs) class. This class is an entry to point for configuring all providers. It is also registered as a *scoped* DI service in the [Startup.cs](./CS/Startup.cs) file.
```
Comment (albertov05) [inline README.md]: 1) `The standard [IHttpContextAccessor](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-context?view=aspnetcore-3.0) with dependency injection allow you to access the HTTP context and pass the retrieved user name to Dashboard providers listed above from the [MultiTenantDashboardConfigurator](./CS/Code/MultiTenantDashboardConfigurator.cs) class.`

Please change "allow" to "allows".

What do you mean by "_from_ the MultiTenantDashboardConfigurator class"?

2) `This class is an entry to point for configuring all providers.`

By "an entry to point" do you mean "an entry point"?

File: README.md [hunk 2]
```diff
@@ -65,17 +65,17 @@ When the application starts, you see the [Index](./CS/Pages/Index.razor) view wh
...
+This example was created based on the **Blazor Server App** Visual Studio template with the *Authentication Type = 'Individual Accounts'* setting (see [Secure ASP.NET Core Blazor Server apps](https://docs.microsoft.com/en-us/aspnet/core/blazor/security/server/?view=aspnetcore-5.0&tabs=visual-studio)). The examples uses the [ASP.NET Core's Identity](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity?view=aspnetcore-5.0&tabs=visual-studio) mechanism for authentication. For this reason, you need to follow the preparation steps below to initialize the Identity database and register required user accounts in it:
```
Comment (albertov05) [inline README.md]:
```suggestion
This example was created based on the **Blazor Server App** Visual Studio template with the *Authentication Type = 'Individual Accounts'* setting (see [Secure ASP.NET Core Blazor Server apps](https://docs.microsoft.com/en-us/aspnet/core/blazor/security/server/?view=aspnetcore-5.0&tabs=visual-studio)). The example uses the [ASP.NET Core's Identity mechanism](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity?view=aspnetcore-5.0&tabs=visual-studio) for authentication. For this reason, you need to follow the steps below to initialize the Identity database and register required user accounts in it:
```

File: README.md [hunk 3]
```diff
@@ -65,17 +65,17 @@ When the application starts, you see the [Index](./CS/Pages/Index.razor) view wh
...
 
-1) Apply the `00000000000000_CreateIdentitySchema` migration to the `aspnet-BlazorAuth-9F8F37E8-C7E8-4D29-BFE6-47204A65FA44` database. You can run the following command in the following command in the Package Manager Console for this purpose:
+1) Apply the `00000000000000_CreateIdentitySchema` migration to the `aspnet-BlazorAuth-9F8F37E8-C7E8-4D29-BFE6-47204A65FA44` database. You can run the following command in the Package Manager Console for this purpose:
 
 ```
 PM> Update-Database
 ```
 
-2) Create and confirm the following user accounts by using the `Identity/Account/Register` form:
+2) Create and confirm the following user accounts in the `Identity/Account/Register` form:
```
Comment (albertov05) [inline README.md:79]: Can you please make the following changes to the lines below?

Line 105:
`Dashboardfor MVC - How to load and save dashboards from/to a database`

Please change "Dashboardfor MVC" to "Dashboard for MVC".

Line 106:
`Dashboardfor MVC - How to load different data based on the current user`

Please change "Dashboardfor MVC" to "Dashboard for MVC".

File: README.md (hunk 1 above)
Comment (natakazakova) [reply README.md]: Thanks, I've updated these lines.

File: README.md (hunk 3 above)
Comment (natakazakova) [reply README.md:79]: Thanks, I've updated.

File: README.md (hunk 1 above)
Comment (albertov05) [reply README.md]: Thank you!

File: README.md (hunk 3 above)
Comment (albertov05) [reply README.md:79]: Thanks.

**Pull request merged**